system-tests/test/browser_crash_handling_spec.js

```diff
@@ -0,0 +1,49 @@
+const systemTests = require('../lib/system-tests').default
+
+describe('Browser Crash Handling', () => {
+  systemTests.setup({
+    settings: {
+      e2e: {},
+    },
+  })
+
+  // It should fail the chrome_tab_crash spec, but the simple spec should run and succeed
+  context('when the tab crashes in chrome', () => {
+    systemTests.it('fails', {
+      browser: 'chrome',
+      spec: 'chrome_tab_crash.cy.js,simple.cy.js',
+      snapshot: true,
+      expectedExitCode: 1,
+    })
+  })
+
+  // It should fail the chrome_tab_crash spec, but the simple spec should run and succeed
+  context('when the tab crashes in electron', () => {
+    systemTests.it('fails', {
+      browser: 'electron',
+      spec: 'chrome_tab_crash.cy.js,simple.cy.js',
+      snapshot: true,
+      expectedExitCode: 1,
+    })
+  })
```

Comment:
```suggestion
  context('when the tab crashes', () => {
    systemTests.it('fails', {
      browser: ['chrome', 'electron']
      spec: 'chrome_tab_crash.cy.js,simple.cy.js',
      snapshot: true,
      expectedExitCode: 1,
    })
  })
```

Reply:
I tried this and it seemed like it got confused and was trying to match the snapshots of the two runs against each other. So i aborted and reverted to the current method.